src/main/java/com/lezurex/overcomplicated/StringUtils.java

```diff
@@ -25,4 +29,171 @@ public static boolean isString(Object possiblySomeStringOrNotQuestionMark) {
+     * @param <PossibleHelloWorldString> The type of the String to check
+     * @return Whether the given String is a Hello World!
+     */
+    public static <PossibleHelloWorldString> boolean isAHelloWorldString(PossibleHelloWorldString possibleHelloWorldString) {
+
+        if (possibleHelloWorldString instanceof final String definitelyAStringButWeDontKnowIfItsAHelloWorld) {
+            if (new Integer(0b1100).equals(definitelyAStringButWeDontKnowIfItsAHelloWorld.length())) {
+                if (0b01001000 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(0)) {
+                    if (0b01100101 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(1)) {
+                        if (0b01101100 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(2)) {
+                            if (0b01101100 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(3)) {
+                                if (0b01101111 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(4)) {
+                                    if (0b00100000 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(5)) {
+                                        if (0b01010111 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(6)) {
+                                            if (0b01101111 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(7)) {
+                                                if (0b01110010 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(8)) {
+                                                    if (0b01101100 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(9)) {
+                                                        if (0b01100100 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(10)) {
+                                                            if (0b00100001 == definitelyAStringButWeDontKnowIfItsAHelloWorld.charAt(11)) {
+                                                                return Boolean.parseBoolean(new String(new byte[] { 0b1110100, 0b1110010, 0b1110101, 0b1100101 }));
+                                                            }
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        return Boolean.parseBoolean(new String(new byte[] { 0b1100110, 0b1100001, 0b1101100, 0b1110011, 0b1100101 }));
+    }
+
+    /**
+     * Does not parse the given Object to a String
+     *
+     * @param objectNotToParse The Object to not parse
+     * @param <ObjectNotToParse> The type of the Object to not parse
+     */
+    public static <ObjectNotToParse> void dontParse(ObjectNotToParse objectNotToParse) {}
```

Comment:
That's some useful piece of code :D

Reply:
Why..?
Do you really want me to test each possible result of the isNotExactlyTwentyOneCharactersLongInALengthRangeOfZeroToThirty and isAHelloWorldString method?